### DISCUSSION TITLE
Have multiple slack teams

### DISCUSSION THREAD

**Discussion by murph12F** [ltworf/localslackirc#436] [Q&A] [answered]
Hello, first of all amazing project, really love it, I want to switch to this instead of using the slack app, but before doing that i d like to know if I can have multiple teams,like I am never of 2 organization in slack any way to have them both and easily switch between them via irc? Just out of curiosity since I haven't tried it yet, what about notifications and calls? If this can also manage calls (even tough I doubt but I d be more than happy to be wrong) I ll permanently delete slack app!

Thx for the time!

**Comment by ltworf** [comment, marked as answer]
You can run multiple instances of this, to join multiple teams. This is very easy to do if you start it via systemd. Just create a new configuration file and follow the instructions within the example configuration to enable the unit.

Calls are not supported and never will be, since it's not something that can be done via IRC.
